### PR TITLE
clarified broker pub/sub

### PR DESCRIPTION
**Pull request by SeanFeldman**
@SimonCropp please review.
Thanks.

### CODE REVIEW

File: nservicebus/messaging/publish-subscribe/index.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ Transport that support native publish–subscribe
 The subscribe workflow for native transports is as follows
 
  1. Subscribers request to the transport the intent to subscribe to certain message types.
- 1. Transport stores both the subscriber names and the message types.
+ 1. Broker stores both the subscription information.
```
Comment (SimonCropp) [inline nservicebus/messaging/publish-subscribe/index.md]: "both the subscription information"  why both if it is a singular?

Comment (SeanFeldman) [inline nservicebus/messaging/publish-subscribe/index.md]: Left from previous statement. Fixed.

File: nservicebus/messaging/publish-subscribe/index.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ Transport that support native publish–subscribe
 The subscribe workflow for native transports is as follows
 
  1. Subscribers request to the transport the intent to subscribe to certain message types.
- 1. Transport stores both the subscriber names and the message types.
+ 1. Broker stores the subscription information.
```
Comment (SimonCropp) [inline nservicebus/messaging/publish-subscribe/index.md]: and why refer to a broker? a broker does not exist in all scenarios

Comment (SeanFeldman) [inline nservicebus/messaging/publish-subscribe/index.md]: It's incorrect to say that the transport is the one that stores subscription info or sends a message to the subscribers. Broker is the one that does it (i.e. non-NSB part).

**Comment by SeanFeldman** [comment]
@SimonCropp should this be merged or there's still some changes to be made?

**Pull request merged**